config/example/config.txt

```diff
@@ -451,3 +451,6 @@ DISABLE_HIGH_POP_MC_MODE_AMOUNT 60
 
 ## Uncomment to give a confirmation before hitting start now
 #START_NOW_CONFIRMATION
+
+##Add a phrase beside it enable Administrative Challenge Phrases
+CHALLENGE_PHRASE
```

Comment:
Shouldn’t this be hashed by default?

Reply:
Yes, it definitely should be hashed by default.

Reply:
it doesn't matter since the value of it is nothing but I can comment it out anyways